test/unit/warp-routes.test.ts

```diff
@@ -19,15 +19,15 @@ describe('Warp Route Configs', () => {
   const routes = localRegistry.getWarpRoutes();
 
   for (const id of Object.keys(routes)) {
-    it(`Route ${id} is valid`, async () => {
+    it(`Core Config ${id} is valid`, async () => {
```

Comment:
Replace Core with WarpCore. Core is too ambiguous with hyperlane core config IMO